tests/ops/api/v1/endpoints/test_storage_endpoints.py

```diff
@@ -293,6 +297,29 @@ def test_patch_storage_config_with_key(
         assert expected_response == response_body
         storage_config.delete(db)
 
+    @pytest.mark.parametrize(
+        "auth_method", [S3AuthMethod.SECRET_KEYS.value, S3AuthMethod.AUTOMATIC.value]
+    )
+    def test_patch_storage_config_with_different_auth_methods(
```

Comment:
It doesn't call out to S3 at all. It does call `initiate_scheduled_request_intake` from `tasks.py` and try to create a scheduler task. I'll patch the call to that function since this test isn't testing task creation.